### ISSUE TITLE
New note dialog behind keyboard

### ISSUE THREAD

**Issue by meichthys** [AndBible/and-bible#2647]
**Describe the bug**
When adding a new note toward the bottom of the screen, the new note dialog is covered by the keyboard.

**Bug was found on AndBible version**
5.0.700.

**Expected behavior**
New note dialog should be visible.

**Screenshots**
![Screenshot_20230618-143313_1.png](https://github.com/AndBible/and-bible/assets/10717998/90c63c6e-043d-4d3a-b76a-784d265099de)

**Smartphone (please complete the following information):**
 - Device: Pixel 7
 - OS: GrapheneOS

**Comment by tuomas2** [comment]
Weird. Can't reproduce on my phone.

**Comment by tuomas2** [comment]
Side note as you seem to use the new beta version and GrapheneOS. Is Google Drive sync feature accessible from GrapheneOS? Have you tried? If it fails, how?

**Comment by meichthys** [comment]
I see the google drive sync option (and it is selected), but I don't see a way to log into google. Perhaps it just uses the account that is signed into the phone? I don't have a google account (part of the reason I use grapheneos), so I can't say for sure if it would work.

I would be greatly interested if there was a Nextcloud sync option, but I understand that the userbase would probably be smaller than google drive users.

**Comment by tuomas2** [comment]
Yep it is only possible to use account that phone has already set up to. 

I am in good hopes that someone will pick up NextCloud sync option. I have a feeling that there might be enough user base of de-googlers among AndBible users. See #2615 for details how to implement it.

**Comment by meichthys** [comment]
I see the issue now. It only happens if the view is split. I have no issues if I only have one Bible open (with no split views).

**Comment by meichthys** [comment]
Can you reproduce the issue if you have the screen split?

**Comment by tuomas2** [comment]
okay I got it reproduced, thanks!

**Comment by tuomas2** [comment]
Closing as fixed by #3572 (commit 7ceb86fc8). The keyboard no longer covers the note dialog on Android 15+ — the WebView area is now manually resized above the keyboard when IME is visible.
